pennylane/_device.py

```diff
@@ -51,7 +51,22 @@ class Device(abc.ABC):
     """
 
     # pylint: disable=too-many-public-methods
-    _capabilities = {}  #: dict[str->*]: plugin capabilities
+    _capabilities = {
+        'model': None,
+        'passthru_interface': None,
+        'supports_reversible_diff': False,
+        'supports_exact': False,
+        'supports_sampled': False,
+        'supports_inverse_operations': False,
+        'supports_tensor_observables': False,
+        'provides_jacobian': False,
+        'executes_in_remote': False,
+        'takes_fixed_number_of_wires': False,
+        'performs_noisy_computation': False,
```

Comment:
Added this last key, because it can be useful for tests. Noise is different to stochasticity, because a circuit that produces a quantum state |01001> will return a deterministic result on a stochastic/sampled simulator, but not on a noisy hardware device or simulator.